src/components/SingleEvent/EventInfo.js

```diff
@@ -142,22 +144,30 @@ const HostUsername = styled('span')`
 class EventInfo extends Component {
   render() {
     const { party, address, className } = this.props
+
+    const admins = extractUsersWithGivenEventRole(party, ROLE.EVENT_ADMIN)
+
     return (
       <EventInfoContainer className={className}>
-        <Date>{party.date || 'Tuesday, 23rd Sep, 2018 9:00 PM'}</Date>
+        <Date>
+          {party.start} - {party.end}
```

Comment:
This is how it looks right now.

![screen shot 2019-01-26 at 13 49 27](https://user-images.githubusercontent.com/2630/51788042-45591d80-2171-11e9-9a9d-c5497f3c2856.png)


1. Can we not display this for old event contract which does not have start and end?
2. Can we format date and time  (something similar to the following, one from google calendar and another from meetup.com).

![screen shot 2019-01-26 at 13 51 54](https://user-images.githubusercontent.com/2630/51788077-ca443700-2171-11e9-9b0a-bf65c9f9a619.png)
![screen shot 2019-01-26 at 13 52 01](https://user-images.githubusercontent.com/2630/51788079-d03a1800-2171-11e9-8b17-db18e3bb16a9.png)

NOTE: If styling is not the focus of this PR, let's hide it completely

Reply:
We can deal with this in another PR - will remove.